datahub-web-react/src/app/ingest/source/builder/RecipeForm/utils.tsx

```diff
@@ -202,6 +208,89 @@ export const REDSHIFT_PASSWORD: RecipeField = {
     rules: null,
 };
 
+export const TABLEAU_CONNECTION_URI: RecipeField = {
+    name: 'connection_uri',
+    label: 'Connection URI',
+    tooltip: 'Tableau host URL.',
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.connection_uri',
+    rules: null,
+};
+
+const tableauProjectFieldPath = 'source.config.projects';
+export const TABLEAU_PROJECT: RecipeField = {
+    name: 'projects',
+    label: 'Projects',
+    tooltip: 'List of projects',
+    type: FieldType.LIST,
+    fieldPath: tableauProjectFieldPath,
+    rules: null,
+    setValueOnRecipeOverride: (recipe: any, values: string[]) =>
+        setListValuesOnRecipe(recipe, values, tableauProjectFieldPath),
+};
+
+export const TABLEAU_SITE: RecipeField = {
+    name: 'site',
+    label: 'Tableau Site',
+    tooltip:
+        'Tableau Site. Always required for Tableau Online. Use empty string to connect with Default site on Tableau Server.',
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.site',
+    rules: null,
+};
+
+export const TABLEAU_USERNAME: RecipeField = {
+    name: 'tableau.username',
+    label: 'Username',
+    tooltip: 'Tableau username, must be set if authenticating using username/password.',
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.username',
+    rules: null,
+};
+
+export const TABLEAU_PASSWORD: RecipeField = {
+    name: 'tableau.password',
+    label: 'Password',
+    tooltip: 'Tableau password, must be set if authenticating using username/password.',
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.password',
+    rules: null,
+};
+
+export const LOOKER_BASE_URL: RecipeField = {
+    name: 'base_url',
+    label: 'Base URL',
+    tooltip: (
+        <TableLineageModeTooltip>
+            <p>
+                Url to your Looker instance: https://company.looker.com:19999 or https://looker.company.com, or similar.
+                Used for making API calls to Looker and constructing clickable dashboard and chart urls.
+            </p>
+        </TableLineageModeTooltip>
+    ),
```

Comment:
since this is just text, you can just pass in the tooltip as a string here instead of wrapping it in react components

Reply:
Checked & Fixed